### PR TITLE
Reforged update 01

### PR DESCRIPTION
**Pull request by manh4wk**
Updated to read replays from Warcraft 3 Reforged v1.32.  Tested on:

v1.32: 1v1 custom game type
v1.32: 4v4 unknown game type
v1.27: 1v1 unknown game type

This is my first PR, let me know if I did something wrong

### CODE REVIEW

File: w3g.py [hunk 1]
```diff
@@ -10,9 +10,10 @@
 import base64
 import zlib
 import struct
+import binascii
 from collections import namedtuple
 
-__version__ = '1.0.2'
+__version__ = '1.0.3'
```
Comment (scopatz) [inline w3g.py]: I'll release a version after this is merged.  No need to change this manually

File: w3g.py [hunk 2]
```diff
@@ -63,6 +64,14 @@ def nulltermstr(b):
         s = b[:i].decode('latin-1')
     return s, i
 
+def fixedlengthstr(b, i):
+    """Returns a string of length i from bytes"""
+    try:
+        s = b[:i].decode('utf-8')
+    except:
```
Comment (scopatz) [inline w3g.py]: This should be `except Exception`

File: w3g.py [hunk 3]
```diff
@@ -1321,6 +1330,35 @@ def from_raw(cls, data):
         kw['raw'] = data[:n]
         return cls(**kw)
 
+class Reforged_Player_Metadata(namedtuple('Reforged_Player_Metadata',
```
Comment (scopatz) [inline w3g.py]: Please remove underscores from class names, ie `ReforgedPlayerMetadata`

**Comment by scopatz** [comment]
Thanks @manh4wk! This is great to see. A couple of small comments inline.  Also, can you please add a news file as per https://regro.github.io/rever-docs/news.html

**Comment by manh4wk** [comment]
I made the updates you requested.

It looks like the WC3 replays will always store the player names as utf-8 and never latin-1 (I don't see how latin-1 would make any sense anyway with their player pool), so I removed that exception after trying to figure out what could make it trigger.  I had been copying what was there for the nulltermstr function.

**Comment by scopatz** [comment]
Thanks!

**Comment by scopatz** [comment]
I'll put out a release now

**Pull request merged**